kontonummer.js

```diff
@@ -33,7 +24,7 @@
             control  : 11
         }
     },{
-        name    : 'Amfa Bank',
+        name    : 'Svea Bank',
         regex   : /^(966[0-9])/,
         modulo  : 11,
         lengths : {
```

Comment:
Shouldn't control be 11 for this bank according to the new docs? Compared with Avanza above they have the same specs

Reply:
You are correct. I missed that this was more than just a renaming of a bank.

Reply:
I've fixed this in the fixup commit c9469c8a966b939e59f89a7288bdd92a0de6b614